proto/ibc/lightclients/tendermint/v1/tendermint.proto

```diff
@@ -75,9 +75,9 @@ message ConsensusState {
   ];
 }
 
-// Misbehaviour is a wrapper over two conflicting Headers
-// that implements Misbehaviour interface expected by ICS-02
-message Misbehaviour {
+// ConflictingHeaders is a wrapper over two conflicting Headers
+// that implements Header interface expected by ICS-02
+message ConflictingHeaders {
```

Comment:
@AdityaSripal thoughts on the naming here?

We could potentially leave this as `Misbehaviour`?

I'm still have some concerns about merging the Misbehaviour and Header interface, mostly around naming. It'd be nice if we could align with the spec as well

We know a normal `Header` can lead to a valid client update, or be evidence of misbehaviour. We also know there can be another structure which can show misbehaviour (in this case duplicate header), but I imagine a light client could have multiple of these structures. My main concern is that `Header` is usually associated with the header produced by a blockchain, but the interface we are using expects potentially one or more headers to be passed in to update the IBC client (either to a valid state or to freeze it)

Reply:
Oh that's great. I'm in favor!